### DISCUSSION TITLE
AOC Day 16: Inception Reference

### DISCUSSION THREAD

**Discussion by rhdeck** [Virtual-Coffee/virtualcoffee.io#781] [Advent of Code 2022]
Discussion of solutions for [Advent of Code Day 16](https://adventofcode.com/2022/day/16) here.

Don't forget about the VC Advent of Code private leaderboard:

Go to [leaderboard/private](https://adventofcode.com/2022/leaderboard/private)
Use join code 363496-e6f42283

**Comment by rhdeck** [comment]
This was very hard for me! Finally got it done by making (what I imagine is kinda like) a graph database. The first part takes 1s, but the second part ran in 18 mins. Maybe not pretty or as efficient as it could be, but I am proud just to keep up. [solve](https://github.com/rhdeck/aoc-2022/blob/main/src/16/day_16.ipynb)

**Comment by doleraj** [comment]
☠ https://github.com/doleraj/adventofcode/blob/main/aoc2022/src/day16/index.ts

that's all I'm gonna day about it
